src/renderer/components/+extensions/__tests__/extensions.test.tsx

```diff
@@ -60,6 +51,9 @@ describe("Extensions", () => {
     di.permitSideEffects(getConfigurationFileModelInjectable);
     di.permitSideEffects(appVersionInjectable);
 
+    mockFetch = fetchMock.sandbox();
```

Comment:
Let's consider if we need a library to mock a low-level and technical abstraction such as `fetch`.

Unit tests would be just as reliable, yet simpler to implement, if instead of `fetch` we had a higher level API, eg. `download` as required here.

Stubbing complex (yet elsewhere unit tested) API's can be a good thing when said complexity is irrelevant in the scope of the unit test.

Reply:
Done